### ISSUE TITLE
Cannot find module 'webpack/lib/RuleSet' in webpack 5

### ISSUE THREAD

**Issue by donggyu04** [dimitor115/bootstrap-vue-loader#51]
I try to use this plugin with webpack5(5.36.2) but, `Cannot find module 'webpack/lib/RuleSet'` error has occered.
Is there any missing or something wrong?

thanks in advance.

- webpack.config.js

```
const BootstrapVueLoader = require('bootstrap-vue-loader');
...

// add BootstrapVueLoader as webpack plugins.
plugins: [
    new BootstrapVueLoader(),
],
```
- error logs
> All files required by `requireStack` already exist in node_modules.
```
[webpack-cli] Failed to load '{project_path}/webpack.config.js' config
[webpack-cli] Error: Cannot find module 'webpack/lib/RuleSet'
Require stack:
- {project_path}/node_modules/bootstrap-vue-loader/lib/plugin.js
- {project_path}/webpack.config.js
- {project_path}/node_modules/webpack-cli/lib/webpack-cli.js
- {project_path}/node_modules/webpack-cli/lib/bootstrap.js
- {project_path}/node_modules/webpack-cli/bin/cli.js
- {project_path}/node_modules/webpack/bin/webpack.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require ({project_path}/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> ({project_path}/node_modules/bootstrap-vue-loader/lib/plugin.js:1:79)
    at Module._compile ({project_path}/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '{project_path}/node_modules/bootstrap-vue-loader/lib/plugin.js',
    '{project_path}/webpack.config.js',
    '{project_path}/node_modules/webpack-cli/lib/webpack-cli.js',
    '{project_path}node_modules/webpack-cli/lib/bootstrap.js',
    '{project_path}/node_modules/webpack-cli/bin/cli.js',
    '{project_path}/node_modules/webpack/bin/webpack.js'
  ]
}
```

**Comment by amokrunner** [comment]
https://github.com/dimitor115/bootstrap-vue-loader/pull/75

**Comment by dimitor115** [comment]
Fixed in 1.0.7 version.
